exec/java-exec/src/main/java/org/apache/drill/exec/planner/logical/DrillSemiJoinRel.java

```diff
@@ -99,8 +102,9 @@ public LogicalOperator implement(DrillImplementor implementor) {
     return new LogicalSemiJoin(leftOp, rightOp, conditions, joinType);
   }
 
-  @Override
-  public boolean isSemiJoin() {
-    return true;
+  @Override public RelOptCost computeSelfCost(RelOptPlanner planner,
+                                              RelMetadataQuery mq) {
+    // REVIEW jvs 9-Apr-2006:  Just for now...
```

Comment:
Please update this comment to describe the reason why this method was overridden.